src/node.cc

```diff
@@ -3059,7 +3059,7 @@ static void DebugPortGetter(Local<Name> property,
 #if HAVE_INSPECTOR
   if (port == 0) {
     Environment* env = Environment::GetCurrent(info);
-    if (env->inspector_agent()->IsStarted())
+    if (env->inspector_agent()->io())
```

Comment:
No. `IsStarted()` should be called `IsInitialized()`, and it always happens if the Agent was constructed, unless I very much misremember. I'll check again, but I'm pretty sure. And I agree, the word "start" in method names is a bit overused in node's c++ source.